src/bindings/c/tests/ov_infer_request_test.cpp

```diff
@@ -353,6 +353,12 @@ TEST_P(ov_infer_request_test, infer_async_wait_for) {
 TEST_P(ov_infer_request_test, infer_async_wait_for_return_busy) {
     OV_EXPECT_OK(ov_infer_request_set_input_tensor_by_index(infer_request, 0, input_tensor));
 
+    ov_callback_t callback;
+    callback.callback_func = [](void* args) {
+        sleep(3);
```

Comment:
In Windows, it should be `Sleep(3)`

Reply:
Maybe use standard sleep `std::this_thread::sleep_for` ?